src/Repository/DataBaseConnection.php

```diff
@@ -1,25 +1,26 @@
 <?php
+
+namespace BookReviews\Repository;
+
+use \PDO;
 /**
- * Created by PhpStorm.
- * User: ioana
- * Date: 11/28/2018
- * Time: 10:06 AM
+ * Class DataBaseConnection
+ * @package BookReviews\Repository
  */
 
```

Comment:
This blank line should be removed as the class comment should always be followed by the class declaration